docker/gcp_pudl_etl.sh

```diff
@@ -95,17 +95,23 @@ ETL_SUCCESS=${PIPESTATUS[0]}
 
 # if pipeline is successful, distribute + publish datasette
 if [[ $ETL_SUCCESS == 0 ]]; then
-    # Dump outputs to s3 bucket if branch is dev or build was triggered by a tag
-    if [ $GITHUB_ACTION_TRIGGER = "push" ] || [ $GITHUB_REF = "dev" ]; then
-        copy_outputs_to_distribution_bucket
-        ETL_SUCCESS=${PIPESTATUS[0]}
-    fi
-
     # Deploy the updated data to datasette
     if [ $GITHUB_REF = "dev" ]; then
         python ~/devtools/datasette/publish.py 2>&1 | tee -a $LOGFILE
         ETL_SUCCESS=${PIPESTATUS[0]}
     fi
+
+    # Compress the SQLite DBs for easier distribution
+    gzip $PUDL_OUTPUT/*.sqlite
```

Comment:
This takes about 5 minutes on my local machine, and reduces the SQLite DBs from 16GB to 4GB total.  It removes the uncompressed files by default.